internal/operators/cnv/config.go

```diff
@@ -0,0 +1,27 @@
+package cnv
+
+import (
+	"strings"
+)
+
+type SupportedGPUsDecoder map[string]bool
+
+type Config struct {
+	// List of supported GPUs: https://issues.redhat.com/browse/CNV-7749
+	SupportedGPUs SupportedGPUsDecoder `envconfig:"CNV_SUPPORTED_GPUS" default:"10de:1db6,10de:1eb8"`
+}
+
+func (d *SupportedGPUsDecoder) Decode(value string) error {
```

Comment:
I don't see this function used anywhere

Reply:
That's a custom decoder used by envconfig (https://github.com/kelseyhightower/envconfig#custom-decoders).